lib/matplotlib/_fontconfig_pattern.py

```diff
@@ -98,11 +96,7 @@ def parse_fontconfig_pattern(pattern):
     for prop in parse.get("properties", []):
         if len(prop) == 1:
             if prop[0] not in _CONSTANTS:
-                _api.warn_deprecated(
-                    "3.7", message=f"Support for unknown constants "
-                    f"({prop[0]!r}) is deprecated since %(since)s and "
-                    f"will be removed %(removal)s.")
-                continue
+                raise ValueError("Support for unknown constants is not supported.")
```

Comment:
```suggestion
                raise ValueError(f"Unknown constants ({prop[0]!r}) are not supported.")
```
I think it's good to tell the user exactly which thing they passed caused the error.

Reply:
Yes, makes sense. Thank you for your input!